rust/index/src/hnsw.rs

```diff
@@ -51,10 +52,23 @@ impl ChromaError for HnswIndexFromSegmentError {
 }
 
 impl HnswIndexConfig {
-    pub fn from_segment(
-        segment: &Segment,
-        persist_path: &std::path::Path,
-    ) -> Result<HnswIndexConfig, Box<HnswIndexFromSegmentError>> {
+    pub fn new_default() -> Self {
```

Comment:
prefer Default trait

https://doc.rust-lang.org/std/default/trait.Default.html

Reply:
Removed it since it's not used anywhere

Reply:
oops, removed now